docs/deploy-development-rp.md

```diff
@@ -176,6 +176,14 @@
   curl -X POST -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/upgrade"
   ```
 
+* Get container logs from an OpenShift pod in a cluster
+  ```bash
+  NAMESPACE=<namespace-name>
+  POD=<pod-name>
+  CONTAINER=<container-name>
+  curl -X POST -k "https://localhost:8443/admin/subscriptions/$AZURE_SUBSCRIPTION_ID/resourceGroups/$RESOURCEGROUP/providers/Microsoft.RedHatOpenShift/openShiftClusters/$CLUSTER/kubernetespodlogs?name=$POD&namespace=$NAMESPACE&container=$CONTAINER"
```

Comment:
The required HTTP method is actually `GET` and not `POST`.

Reply:
Ohh .. my bad.. very bad .. will correct it ..